doc/Command-Reference.md

```diff
@@ -718,6 +718,33 @@ This command displays the cause of the previous reboot
   User issued reboot command [User: admin, Time: Mon Mar 25 01:02:03 UTC 2019]
   ```
 
+```
+Note: The CLI extensions shown in this block are applicable only to smartswitch platforms. When these extensions are used on a regular switch the extension will be ignored and the output will be the same irrespective of the options.
+
+CLI Extensions Applicable to Smartswtich
+  - show reboot-cause all
+  - show reboot-cause history all
+  - show reboot-cause history DPUx
+  - show reboot-cause history SWITCH
```

Comment:
@rameshraghupathy can we keep this as same to "show reboot-cause history" i.e it will show only the reboot cause of NPU or the Switch. So no need for SWITCH option.

Reply:
@prgeor "show reboot-cause history" also exists as it has been always.  In the extended option for smartswitch, if the user wants to print only the NPU/SWITCH reboot-cause history we have the SWITCH option.  If this is not needed I can clean it.

Reply:
@prgeor removed the SWITCH option